openmetadata-ui/src/main/resources/ui/src/pages/TagPage/TagPage.tsx

```diff
@@ -0,0 +1,579 @@
+    return tagItem
+      ? [
+          {
+            name: 'Classifications',
+            url: ROUTES.TAGS,
+            activeTitle: false,
+          },
+          {
+            name: tagItem.classification?.name ?? '',
+            url: tagItem.classification?.fullyQualifiedName
+              ? getClassificationDetailsPath(
+                  tagItem.classification.fullyQualifiedName
+                )
+              : '',
+            activeTitle: false,
+          },
+        ]
+      : [];
+  }, [tagItem]);
+
+  const handleAssetClick = useCallback(
+    (asset?: EntityDetailsObjectInterface) => {
+      setPreviewAsset(asset);
+    },
+    []
+  );
+
+  const hasEditPermission = useMemo(() => !tagItem?.disabled, [tagItem]);
+
+  const onDescriptionUpdate = async (updatedHTML?: string) => {
+    if (tagItem) {
+      if (tagItem.description !== updatedHTML) {
+        const updatedTableDetails = {
+          ...tagItem,
+          description: updatedHTML,
+        };
+        const jsonPatch = compare(tagItem, updatedTableDetails);
+        try {
+          const response = await patchTag(tagItem.id ?? '', jsonPatch);
+
+          setTagItem(response);
+
+          if (tagItem?.name !== updatedTableDetails.name) {
+            history.push(
+              getClassificationTagPath(response.fullyQualifiedName ?? '')
+            );
+          }
+        } catch (error) {
+          showErrorToast(error as AxiosError);
+        }
+        setIsDescriptionEditable(false);
+      } else {
+        setIsDescriptionEditable(false);
+      }
+    }
+  };
+
+  const getTagData = async () => {
+    try {
+      setIsLoading(true);
+      if (tagFqn) {
+        const response = await getTagByFqn(tagFqn);
+        setTagItem(response);
+      }
+    } catch (e) {
+      if (e instanceof AxiosError) {
+        showErrorToast(e);
+      }
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const activeTabHandler = (tab: string) => {
+    if (tagItem) {
+      history.push({
+        pathname: getClassificationTagPath(
+          tagItem.fullyQualifiedName ?? '',
+          tab
+        ),
+      });
+    }
+  };
+
+  const editDescriptionPermission = useMemo(() => {
+    const editDescription = checkPermission(
+      Operation.EditDescription,
+      resourceType,
+      permissions
+    );
+
+    return editDescription;
+  }, [permissions, resourceType]);
+
+  const updateTag = async (updatedData: Tag) => {
+    if (tagItem) {
+      const jsonPatch = compare(tagItem, updatedData);
+
+      try {
+        const response = await patchTag(tagItem.id ?? '', jsonPatch);
+
+        setTagItem(response);
+
+        if (tagItem?.name !== updatedData.name) {
+          history.push(
+            getClassificationTagPath(response.fullyQualifiedName ?? '')
+          );
+        }
+      } catch (error) {
+        showErrorToast(error as AxiosError);
+      }
+    }
+  };
```

Comment:
same condition of name is not required. Check if you can combine both onDescriptionUpdate and this method.